scripts/qesap/test/e2e/test.sh

```diff
@@ -21,43 +21,194 @@ rm -rf "${QESAPROOT}"
+echo "#######################################################################"
+echo "###                                                                 ###"
+echo "###                         A N S I B L E                           ###"
+echo "###                                                                 ###"
+echo "#######################################################################"
+test_step "[test_3.yaml] Run Ansible with no playbooks"
 # Keep in mind that test_3.yaml has no playbooks at all
 touch "${TEST_PROVIDER}/inventory.yaml"
 qesap.py --verbose -b ${QESAPROOT} -c test_3.yaml ansible || test_die "test_3.yaml fail on ansible"
 
-test_step "[test_4.yaml] Run Ansible create"
-touch "${QESAPROOT}/ansible/playbooks/sambuconero.yaml"
+test_step "[test_4.yaml] Run Ansible dryrun"
+cp sambuconero.yaml "${QESAPROOT}/ansible/playbooks/sambuconero.yaml"
 cp inventory.yaml "${TEST_PROVIDER}/inventory.yaml"
 qesap.py --verbose -b ${QESAPROOT} -c test_4.yaml configure || test_die "test_4.yaml fail on configure"
+# At the moment e2e does not ahve a way to really run ansible
 qesap.py --verbose -b ${QESAPROOT} -c test_4.yaml --dryrun ansible || test_die "test_4.yaml fail on ansible"
-qesap.py --verbose -b ${QESAPROOT} -c test_4.yaml --dryrun ansible | tee "${QESAPROOT}/ansible.log"
+qesap.py --verbose -b ${QESAPROOT} -c test_4.yaml --dryrun ansible |& tee "${QESAPROOT}/ansible.log"
 grep -E "ansible.*-i.*fragola/inventory.yaml.*all.*ssh-extra-args" "${QESAPROOT}/ansible.log" || test_die "test_4.yaml dryrun fails in ansible command"
-grep -E "ansible-playbook.*-i.*fragola/inventory.yaml.*ansible/playbooks/sambuconero.yaml" "${QESAPROOT}/ansible.log" || test_die "test_4.yaml drayrun fails in ansible-playbook command"
+grep -E "ansible-playbook.*-i.*fragola/inventory.yaml.*ansible/playbooks/sambuconero.yaml" "${QESAPROOT}/ansible.log" || test_die "test_4.yaml dryrun fails in ansible-playbook command"
+rm "${QESAPROOT}/ansible.log"
+
+test_step "[test_4.yaml] Run Ansible PASS"
+qesap.py --verbose -b ${QESAPROOT} -c test_4.yaml ansible || test_die "test_4.yaml fail on ansible"
+
+test_step "[test_4.yaml] Ansible PASS verbosity"
+rm "${QESAPROOT}/test_4_ansible_pass_verbose.txt" || echo "No ${QESAPROOT}/test_4_ansible_pass_verbose.txt"
+rm "${QESAPROOT}/test_4_ansible_pass.txt" || echo "No ${QESAPROOT}/test_4_ansible_pass.txt"
+set +e
+qesap.py -b ${QESAPROOT} -c test_4.yaml ansible |& tee "${QESAPROOT}/test_4_ansible_pass.txt"
+qesap.py --verbose -b ${QESAPROOT} -c test_4.yaml ansible |& tee "${QESAPROOT}/test_4_ansible_pass_verbose.txt"
+
+grep -qE "^DEBUG" "${QESAPROOT}/test_4_ansible_pass_verbose.txt"
+rc=$?
+if [ $rc -ne 0 ]; then
+  test_die "rc:$rc in verbose mode there should be some DEBUG"
+fi
+grep -qE "^INFO" "${QESAPROOT}/test_4_ansible_pass_verbose.txt"
+rc=$?
+if [ $rc -ne 0 ]; then
+  test_die "rc:$rc in verbose mode there should be some INFO"
+fi
+grep -E "TASK \[Say hello\]" "${QESAPROOT}/test_4_ansible_pass_verbose.txt" | wc -l
```

Comment:
this test is intentionally left uncompleted. The expected value should be 1 but, due to a bug in the glue script that has to be fixed, is now 2

Reply:
I'll convert it to a real test in a next PR where I'll also at the same time fix the bug.